### ISSUE TITLE
Add method to send message with attributes

### ISSUE THREAD

**Issue by bradbyte** [ccm-innovation/react-native-twilio-chat#6]
See https://github.com/ccm-innovation/react-native-twilio-ip-messaging/issues/74.

Basically, we need to expose a method to construct a new message instance and then send that inside the `sendMessage` call. That way, you don't have to send message and then immediately update with attributes.

**Comment by svlaev** [comment]
@bradbumbalough we have such functionality: https://github.com/ccm-innovation/react-native-twilio-chat/pull/7
Usage is as simple as:
```
await this.channel.sendAttributedMessage(text, {
  'foo': 'bar',
  'baz': 'qux'
});
```

**Comment by bradbyte** [comment]
@svlaev thanks! yeah I forgot to cross reference the issue with your PR.

**Comment by bradbyte** [comment]
Fixed in #17.

**Comment by bradbyte** [comment]
Released in [`0.2.0`](https://github.com/ccm-innovation/react-native-twilio-chat/releases/tag/v0.2.0).
